test/test-signal.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// Testcase to produce NodeReport on signal
+if (process.argv[2] === 'child') {
+  require('../').setEvents("signal");
+
+  function myLoop() {
+  var list = [];
+    for (var i=0; i<10000000000; i++) {
```

Comment:
there are some indentations problems here

what are you trying to do? are you attempting to show that signals interrupt a js busy-loop?

I think an interesting test would be having the child also listen on SIGUSR1, and show that both nodereport and the child get the signal (I assume they should, but nodereport may have stolen the signal from uv or node, good to show it did not)